src/BaseWrapper.sol

```diff
@@ -46,10 +61,12 @@ abstract contract BaseWrapper is IERC7399 {
         return result;
     }
 
-    /// @dev Call the flashloan function in the child contract
+    /// @dev Call the legacy flashloan function in the child contract. This is where we borrow from Aave, Uniswap, etc.
     function _flashLoan(address asset, uint256 amount, bytes memory params) internal virtual;
 
-    /// @dev Handle the common parts of bridging the callback
+    /// @dev Handle the common parts of bridging the callback from legacy to ERC7399. Transfer the funds to the loan
+    /// receiver. Call the callback supplied by the original borrower. Approve the repayment if necessary. If there is
+    /// any result, it is kept in a storage variable to be retrieved on `flash` after the legacy flash loan is finished.
     function bridgeToCallback(address asset, uint256 amount, uint256 fee, bytes memory params) internal {
```

Comment:
hmm, this function name should start with a `_`

Reply:
Fixed in #14